<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GobangForm</class>
 <widget class="QWidget" name="GobangForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GobangForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="grid_chess"/>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="5" column="0">
       <widget class="QLabel" name="label_white">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string>白子</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_start">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_peace">
        <property name="text">
         <string>求和</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_regret">
        <property name="text">
         <string>悔棋</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_black">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string>黑子</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btn_giveup">
        <property name="text">
         <string>认负</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
